packages/next/shared/lib/image-config.ts

```diff
@@ -74,6 +74,9 @@ export type ImageConfigComplete = {
 
   /** @see [Dangerously Allow SVG](https://nextjs.org/docs/api-reference/next/image#dangerously-allow-svg) */
   contentSecurityPolicy: string
+
+  /** @see [unoptimized](https://nextjs.org/docs/api-reference/next/image#unoptimized) */
+  unoptimized: boolean | undefined
```

Comment:
Do we want to start with having this under `experimental.images` so that we can iterate on it while testing?

Reply:
Perhaps. I'm not sure what we would change since its just a pass-through. Perhaps the combination of loader might affect it. Originally I had this as `loader: unoptimized` but it felt better to do `unoptimized: true` to match the prop and made docs much easier. So maybe I answered my own question that we might want to change it to a loader 😆

Reply:
I moved it to experimental 👍